breathecode/admissions/views.py

```diff
@@ -877,6 +877,13 @@ def post(self, request, academy_id=None):
         if request.data.get('current_day'):
             raise ValidationException('current_day field is not allowed', slug='current-day-not-allowed')
 
+        language = request.data.get('language')
+        if language is not None:
+            if type(language) == str:
+                request.data['language'] = language.lower()
+            else:
```

Comment:
Actually exists cohorts with the language in uppercase, we need more changes to prevent possible failures